app/src/forms/admin/routes.js

```diff
@@ -42,21 +195,193 @@ routes.get('/forms/:formId/versions/:formVersionId', async (req, res, next) => {
   await controller.readVersion(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/forms/{formId}/formUsers:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    summary: Get the list of users for the active forms in the system
+ *    description: This endpoint will fetch all the users added as team members to this form and their roles/permissions.
+ *    security:
+ *      - openId: []
+ *    parameters:
+ *      - in: path
+ *        name: formId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: ID of the form.
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminGetFormUserRoles'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ *      '404':
+ *        $ref: '#/components/responses/Error/ResourceNotFound'
+ * */
 routes.get('/forms/:formId/formUsers', async (req, res, next) => {
   await controller.getFormUserRoles(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/forms/{formId}/addUser:
+ *  put:
```

Comment:
I'm a bit confused here: is the formId actually "required" both as a path parameter and as a query parameter? And should "role" be listed as a non-required query parameter?

Reply:
I was too but that was from the showcase team. It is part of what we need clean up or fix

Reply:
Thank you!